Makefile

```diff
@@ -9,10 +9,10 @@ test:
 	pytest --cov samtranslator --cov-report term-missing --cov-fail-under 95 tests
 
 black:
-	black setup.py samtranslator/* tests/* bin/*.py
+	black setup.py samtranslator/**/*.py tests/**/*.py bin/*.py
```

Comment:
Why is this change necessary? This also assumes the shell supports `**` (which isn't in POSIX `sh`), which e.g. in Bash is only enabled if `globstar` is set. AFAIK `make` won't necessarily default to Bash.

Tangentially, can we instead simplify this to `black setup.py samtranslator tests bin` and change include/exclude behavior in the config?

Reply:
I intended to skip checking the .pyc files. But looking at the doc, yeah you are right, it's not necessary.
Will revert this commit